packages/hoppscotch-backend/src/user/user.service.ts

```diff
@@ -41,12 +41,8 @@ export class UserService {
 
     return {
       ...dbUser,
-      currentRESTSession: dbCurrentRESTSession
-        ? JSON.stringify(dbCurrentRESTSession)
-        : null,
-      currentGQLSession: dbCurrentGQLSession
-        ? JSON.stringify(dbCurrentGQLSession)
-        : null,
+      currentRESTSession: JSON.stringify(dbCurrentRESTSession) ?? null,
+      currentGQLSession:  JSON.stringify(dbCurrentGQLSession) ?? null,
```

Comment:
`JSON.stringify(null)` output for this will be `"null"`. You have to check if the value is null or not in the first place. And then stringify later

Reply:
@mir-ishfaq-hussain Please revert these changes to the old code.

Reply:
@balub i have done the required changes